docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -299,19 +299,27 @@ Update of the software will be managed by the swupd tool::
 Installing Ansible with ``pip``
 --------------------------------
 
-Ansible can be installed with ``pip``, the Python package manager.  If ``pip`` isn't already available on your system of Python, run the following commands to install it::
+Ansible can be installed with ``pip``, the Python package manager. It should be noted that macOS requires a slightly different use of ``pip`` than *nix due to ``openssl`` requirements, therefore pip must be run as a module.  If ``pip`` isn't already available on your system of Python, run the following commands to install it::
```

Comment:
The `*` in restrectured text indicates bold text. `*nix` looks like a bold indicator that is missing the closing `*`, which is causing the test failure.
```suggestion
Ansible can be installed with ``pip``, the Python package manager. It should be noted that macOS requires a slightly different use of ``pip`` than ``*nix`` due to ``openssl`` requirements, therefore pip must be run as a module.  If ``pip`` isn't already available on your system of Python, run the following commands to install it::
```

Reply:
Are there docs from Apple somewhere describing this requirement? Is this trying to use `/usr/bin/python` on macOS rather than Python from Homebrew or `pyenv`? Just trying to understand the context a bit more here since `python` on macOS is terribly ambiguous.

I use custom `CFLAGS` and `LDFLAGS` when compiling Python on macOS to use the appropriate `openssl` libraries. Is this a way to avoid having to do that by using `/usr/bin/python` that ships with macOS?